src/components/DayPickerRangeController.jsx

```diff
@@ -299,22 +309,28 @@ export default class DayPickerRangeController extends React.Component {
             modifiers = this.deleteModifier(modifiers, momentObj, 'blocked');
           }
 
-          if (isOutsideRange(momentObj)) {
-            modifiers = this.addModifier(modifiers, momentObj, 'blocked-out-of-range');
-          } else {
-            modifiers = this.deleteModifier(modifiers, momentObj, 'blocked-out-of-range');
+          if (recomputeOutsideRange) {
```

Comment:
✅ &nbsp;Fixed (hoping the superfluous merge commit cece6c0 doesn't screw anything up).